src/operator/regression_output.cc

```diff
@@ -90,7 +90,7 @@ The parameter `grad_scale` can be used to change this scale to `grad_scale/m`.
 )code" ADD_FILELINE);
 
 MXNET_OPERATOR_REGISTER_REGRESSION_BWD(_backward_linear_reg_out, mshadow_op::minus, true);
-
+/*
```

Comment:
Will this code be re-enabled before merge?

Reply:
Of course. I am just removing some instantiated templates and try to see what causes this error. Seems that I should divide three regression operators in different files (not sure yet).